index.html

```diff
@@ -0,0 +1,476 @@
+                <div class="FIAuthor">
+                    <p class="FIP FIAPadding">by Yessi Bello Perez </p>
+                    <i class="fa fa-clock-o FIAPadding"></i>
+                    <p class="FIP FIAPadding">42 minutes ago</p>
+                    <i class="fa fa-share-alt FIAPadding"></i>
+                    <p class="FIP FIAPadding">0</p> 
+                </div>
+            </article>
+            <article id="secondInsight">
+                <h2 class="titleInsight">The tech media keeps hyping awful, exploitative fintech products</h2>
+                <div class="authorLeft">
+                    <p class="FIP FIAPadding">by Matthew Hughes</p>
+                    <i class="fa fa-clock-o FIAPadding"></i>
+                    <p class="FIP FIAPadding">58 minutes ago</p>
+                    <i class="fa fa-share-alt FIAPadding"></i>
+                    <p class="FIP FIAPadding">0</p> 
+                </div>
+            </article>
+            <article id="thirdInsight">
+                <h2 class="titleInsight">Video: Burn in hell, Twitter — a guide on deleting or deactivating your account</h2>
+                <div class="authorLeft">
+                    <p class="FIP FIAPadding">by Callum Booth</p>
+                    <i class="fa fa-clock-o FIAPadding"></i>
+                    <p class="FIP FIAPadding">2 hours ago</p>
+                    <i class="fa fa-share-alt FIAPadding"></i>
+                    <p class="FIP FIAPadding">0</p> 
+                </div>
+            </article>
+        </section>
+    <main>
+        <section class="latestNews">
+            <h2>Latest news</h2>
+            <figure class="latestNewsFig">
+                <img src="images/FBNEWS.jpg" alt="image">
+                <figcaption>
+                    <p class="latestNewsTitle"><strong>How to use Google to find podcast episodes on a specific topic</strong></p>
+                    <p class="latestNewsAuthor">Mix - 29 minutes ago</p>
+                </figcaption>
+            </figure>
+            <figure class="latestNewsFig">
+                <img src="images/location.jpg" alt="image">
+                <figcaption>
+                    <p class="latestNewsTitle"><strong>How to use Google to find podcast episodes on a specific topic</strong></p>
+                    <p class="latestNewsAuthor">Mix - 29 minutes ago</p>
+                </figcaption>
+            </figure>
+            <figure class="latestNewsFig">
+                    <img src="images/president.jpg" alt="image">
+                    <figcaption>
+                        <p class="latestNewsTitle"><strong>How to use Google to find podcast episodes on a specific topic</strong></p>
+                        <p class="latestNewsAuthor">Mix - 29 minutes ago</p>
+                    </figcaption>
+                </figure>
+                <figure class="latestNewsFig">
+                    <img src="images/iphone.jpg" alt="image">
```

Comment:
#### `HINT`

This image file seems to be missing even though I see it on the readme screenshot. :thinking:  See below:

![missing image](https://user-images.githubusercontent.com/36057474/62815337-ca08f900-bb0f-11e9-98d1-7c9ec0044dd2.png)